mods/ra/rules/vehicles.yaml

```diff
@@ -691,7 +691,10 @@ DTRK:
 	Explodes:
 		Weapon: MiniNuke
 		EmptyWeapon: MiniNuke
-	DemoTruck:
+	Detonates:
+		ChargeDelay: 0
```

Comment:
Why do you specify the ChargeDelay and DetonationDelay here, when the default is also 0?